docs/resources/cluster.md

```diff
@@ -1,73 +1,16 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "pcluster_cluster Resource - pcluster"
+page_title: "aws-parallelcluster_cluster Resource - aws-parallelcluster"
 subcategory: ""
 description: |-
   Create a managed cluster in an AWS Region.
 ---
 
-# pcluster_cluster (Resource)
+# aws-parallelcluster_cluster (Resource)
 
 Create a managed cluster in an AWS Region.
 
-## Example Usage
 
```

Comment:
Why remove the example?

Reply:
docs folder is auto-generated by the terraform tool. Let me check...

Reply:
Oh I see why, iterating, thanks for the heads up

Reply:
Solved in 2nd iteration